Source/AtomicJS/Javascript/JSAPI.cpp

```diff
@@ -69,13 +69,17 @@ void js_class_declare_internal(JSVM* vm, void* uniqueClassID, const char* packag
         duk_put_prop(ctx, -3);
         duk_pop(ctx);
     }
-    else
+    
+    if (!duk_get_global_string(ctx, package))
     {
-        // RefCounted only supported in Atomic package
-        assert(String("Atomic") == package );
+        // Failure leaves undefined on the stack
```

Comment:
Does this code path ever get hit?  The package object should exist before declaring any classes in it.  The assert that was in here when uniqueID == NULL is due to how the class -> package lookup is done when pushing a variant (to only push bound classes).  

I made a ticket to improve this:  https://github.com/AtomicGameEngine/AtomicGameEngine/issues/124

Were you hitting the assert binding something that was derived directly from RefCounted?